Components/FlightControl/RocketSM.cpp

```diff
@@ -1013,10 +1010,8 @@ RocketState Test::HandleCommand(Command& cm)
             MEVManager::CloseMEV();
             break;
         case RSC_TEST_MEV_ENABLE:
-            GPIO::MEV_EN::On();
             break;
         case RSC_TEST_MEV_DISABLE:
-            GPIO::MEV_EN::Off();
             break;
```

Comment:
Might as well just delete both cases.

Reply:
This is one I don't think we should delete, cause it let's us explicitly know what should be allowed in this state in case we decide a logical interlock is a good idea

Reply:
Ahh ok that makes sense then.